docs/91_version_history.rst

```diff
@@ -26,38 +26,56 @@ FogLAMP v1
 ==========
 
 
-v1.0
+v1.2
 ----
 
-Release Date: 2017-12-11
+Release Date: 2018-04-23
 
 
-Features
-~~~~~~~~
+New Features
+~~~~~~~~~~~~
 
-- All the essential microservices are now in place: *Core, Storage, South, North*.
-- Storage plugins available in the main repository:
+- **Changes in the REST API**
 
-  - **Postgres**: The storage layer relies on PostgreSQL for data and metadata
+  - **ping Method**: the ping method now returns uptime, number of records read/sent/purged and if FogLAMP requires REST API authentication.
 
-- South plugins available in the main repository:
+- **Storage Layer
 
-  - **CoAP Listener**: A CoAP microservice plugin listening to client applications that send data to FogLAMP
+  - **Default Storage Engine**: The default storage engine is now SQLite. We provide a script to migrate from PostgreSQL in 1.1.1 version to 1.2. PostgreSQL is still available in the main repository and package, but it will be removed in future versions. 
```

Comment:
Is it more accurate to say "it will be moved to a operate repository" rather than it will be removed?